features/step_definitions/git-flow.js

```diff
@@ -157,7 +157,7 @@ function waitForBuild(screwdriverInstance, desiredSha) {
 module.exports = function server() {
     // eslint-disable-next-line new-cap
     this.Before(() => {
-        this.instance = 'http://api.screwdriver.cd';
```

Comment:
Had to. Its `Before` applies to all the scenarios. We could move this whole block to `before-hooks` but didn't want to mess with git-flow